src/gt4py/next/common.py

```diff
@@ -1073,4 +1074,8 @@ def register_builtin_func(
     @classmethod
     def __gt_builtin_func__(cls, /, func: fbuiltins.BuiltInFunction[_R, _P]) -> Callable[_P, _R]:
         return cls._builtin_func_map.get(func, NotImplemented)
-        return cls._builtin_func_map.get(func, NotImplemented)
+
+
+#: Numeric value used to represent missing values in connectivities.
+#: Adopted from UGRID Conventions (http://ugrid-conventions.github.io/ugrid-conventions/)
+SKIP_VALUE: Final[int] = -1
```

Comment:
Decided for this name as everyone is familiar already and the private `_FillValue` doesn't really convince me.